clusters.py

```diff
@@ -210,120 +210,131 @@ def HLossClassifier(df: pd.DataFrame) -> pd.DataFrame:
     df: dataframe with HL classes as a new column, with several new working
       columna added.
   """
-
-  #HFPTA
-  df['R_HFPTA'] = df[['R1000', 'R2000', 'R4000']].mean(axis=1)
-  df['L_HFPTA'] = df[['L1000', 'L2000', 'L4000']].mean(axis=1)
-
-  # PTA - 500, 1000, 2000
-  df['R_PTA'] = df[['R500', 'R1000', 'R2000']].mean(axis=1)
-  df['L_PTA'] = df[['L500', 'L1000', 'L2000']].mean(axis=1)
-
-  #PTA all -  500,1000,2000,4000
-  df['R_PTA_All'] = df[['R500', 'R1000', 'R2000', 'R4000']].mean(axis=1)
-  df['L_PTA_All'] = df[['L500', 'L1000', 'L2000', 'L4000']].mean(axis=1)
- 
-  #LFPTA -  250, 500,1000
-  df['R_LFPTA'] = df[['R500', 'R1000']].mean(axis=1)
-  df['L_LFPTA'] = df[['L500', 'L1000']].mean(axis=1)
-
-  #UHFPTA - 2000, 4000, 80000
-  df['R_UHFPTA'] = df[['R2000', 'R4000', 'R8000']].mean(axis=1)
-  df['L_UHFPTA'] = df[['L2000', 'L4000', 'L8000']].mean(axis=1)
- 
-  #PT Bone conduction modeled
-  df['R_PTA_BC_Mod'] = df[['RBone500', 'RBone1000', 'RBone2000']].mean(axis=1)
-  df['L_PTA_BC_Mod'] = df[['LBone500', 'LBone1000', 'LBone2000']].mean(axis=1)
-
-  #HFPTA Bone conduction modeled
-  df['R_HFPTA_BC_Mod'] = df[['RBone1000', 'RBone2000','RBone4000']].mean(axis=1)
-  df['L_HFPTA_BC_Mod'] = df[['LBone1000', 'LBone2000','LBone4000']].mean(axis=1)
-
-  #BC average of 500, 1, 2, 4kHz
-  df['R_PTA_BC_All'] = df[['RBone500', 'RBone1000',
-                           'RBone2000', 'RBone4000']].mean(axis=1)
-  df['L_PTA_BC_All'] = df[['LBone500', 'LBone1000',
-                           'LBone2000', 'LBone4000']].mean(axis=1)
+  
+### Having BC thresholds in both ears in cases of symmetric hearing loss 
+
+  frequencies = ['250', '500', '1000', '2000', '4000']
+  
+  # Loop through each frequency
+  for freq in frequencies:
```

Comment:
This can be written in a more python way as a list comprehension
right_col = [f'RBone{freq} for freq in frequencies]
left_col = ...